bears/actions/AddNewlineAction.py

```diff
@@ -0,0 +1,28 @@
+from coalib.misc.Shell import run_shell_command
+from coalib.results.result_actions.ResultAction import ResultAction
+
+
+class AddNewlineAction(ResultAction):
+
+    SUCCESS_MESSAGE = 'New Line added successfully.'
+
+    def __init__(self, shortlog, body):
+        self.shortlog = shortlog
+        self.body = body
+
+    @staticmethod
+    def is_applicable(result,
+                      original_file_dict,
+                      file_diff_dict,
+                      applied_actions=()):
+        return 'AddNewlineAction' not in file_diff_dict
+
+    def apply(self, result, original_file_dict, file_diff_dict):
+        """
+        Add (N)ewline
+        """
+        new_commit_message = '{}\n\n{}'.format(self.shortlog, self.body)
+        command = 'git commit --amend -m "{}"'.format(new_commit_message)
```

Comment:
@abhishalya I think `git commit --amend` also adds the stages changes. This is not good. Any alternatives?

Reply:
I guess `git rebase` would do then.

Reply:
Have a look at [this]. Even rebase is not ideal if someone is using your commits.

Reply:
ping @Vamshi99 @kriti21

Reply:
Yeah, but here we are expecting the user to change the commit message. If the commit is not pushed doing rebase is just fine.

Reply:
Maybe we can warn that history will be rewritten action is applied.

Reply:
Yeah, a warning message would be good 👍

Reply:
> I guess `git rebase` would do then.

`git rebase` doesn't run when there are unstaged or uncommitted changes.

Reply:
`git commit --amend -o` will do the job.